profiles/commands.py

```diff
@@ -23,8 +24,8 @@ def extract_email_addresses(orcid_record: dict, only_verified: bool = True) -> L
 
 def _update_name_from_orcid_record(profile: Profile, orcid_record: dict) -> None:
     if 'name' in orcid_record.get('person', {}):
-        given_name = orcid_record['person']['name'].get('given-names', {}).get('value')
-        family_name = orcid_record['person']['name'].get('family-name', {}).get('value')
+        given_name = jmespath.search('person.name."given-names".value', orcid_record)
+        family_name = jmespath.search('person.name."family-name".value', orcid_record)
```

Comment:
needs 3 test cases:

- missing `given-names` and/or `family-name` fields
- fields are present, but with value `null`
- fields are present, but with value `{"value":""}`